frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -100,22 +100,30 @@ const ProductAnalyticsOnboarding = (): JSX.Element => {
 }
 const SessionReplayOnboarding = (): JSX.Element => {
     const { hasAvailableFeature } = useValues(userLogic)
+    const { currentTeam } = useValues(teamLogic)
+
     const configOptions: ProductConfigOption[] = [
         {
             type: 'toggle',
             title: 'Capture console logs',
             description: `Capture console logs as a part of user session recordings. 
                             Use the console logs alongside recordings to debug any issues with your app.`,
             teamProperty: 'capture_console_log_opt_in',
-            value: true,
+            value:
+                typeof currentTeam?.capture_console_log_opt_in === 'boolean'
```

Comment:
Technically according to our types this field is always a boolean, so we should update that as well.